x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group1/aggregate.ts

```diff
@@ -6,10 +6,13 @@
  */
```

Comment:
Can you add this test below? To show that you can not bypass the field attribute in aggregation
```ts
it('should reject incorrect aggregation without using field attributes', async () => {
      const invalidFieldAggs = JSON.stringify({
        outcome: {
          terms: 'alert.attributes.lastRun.outcome',
        },
      });

      const response = await supertest
        .post(`${getUrlPrefix(Spaces.space1.id)}/internal/alerting/rules/_aggregate`)
        .set('kbn-xsrf', 'foo')
        .send({
          aggs: invalidFieldAggs,
        });

      expect(response.status).to.eql(400);
      expect(response.body.message).to.eql(
        'Invalid aggregation: [outcome.terms]: could not parse object value from json input: Bad Request'
      );
    });
```

Reply:
@kobelb ^

Reply:
Test no longer applicable since we do not apply arbitrary aggregations.